### ISSUE TITLE
Exception when going back to previous page

### ISSUE THREAD

**Issue by tuanp** [diegoveloper/flutter_keyboard_actions#26]
Hi,
When the input is in editing and keyboard is showing, if I click Back button and going back to the previous page, I got this exception. Could you help to check it?

```
flutter: When the exception was thrown, this was the stack:
flutter: #0      State.setState.<anonymous closure> (package:flutter/src/widgets/framework.dart:1090:9)
flutter: #1      State.setState (package:flutter/src/widgets/framework.dart:1116:6)
flutter: #2      FormKeyboardActionState._updateOffset (package:keyboard_actions/keyboard_actions.dart:304:7)
flutter: #3      FormKeyboardActionState._focusChanged.<anonymous closure> (package:keyboard_actions/keyboard_actions.dart:242:7)
flutter: #4      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1012:15)
flutter: #5      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:960:9)
flutter: #6      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:864:5)
flutter: #10     _invoke (dart:ui/hooks.dart:219:10)
flutter: #11     _drawFrame (dart:ui/hooks.dart:178:3)
flutter: (elided 3 frames from package dart:async)
```

**Comment by tuanp** [comment]
@diegoveloper Pls help to check mounted status when updating.

**Comment by diegoveloper** [comment]
Fixed on #27
